.env.example

```diff
@@ -1,3 +1,6 @@
+# Copy this to `.env` and `.env.test` files
+
 SENTRY_AUTH_TOKEN=
+EXPO_PUBLIC_ENV=development
 EXPO_PUBLIC_LOG_LEVEL=debug
 EXPO_PUBLIC_LOG_DEBUG=
```

Comment:
So one env change here is: Expo doesn't actually use `NODE_ENV`, instead it uses that value to load a `.env.*` file. E.g. `NODE_ENV=test` loads `.env.test`.